README.md

```diff
@@ -1,35 +1,53 @@
- 1. This is the License Agreement form. Accept the agreement to proceed ahead. ![License Agreement](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/License%20Agreement.png "License Agreement Form")
- 2. Once you accept the agreement, you are presented with the below screen. Browse or type the location of the folder where you would like to install Exercism. ![Install Location](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/Install%20Location.png "Install Location Form")
- 3. After this, there might be a pause. Don't worry everything is working fine and you would soon be presented with the below screen where the Setup downloads the latest version of Exercism CLI from it's Github Repository - ![Client Download Form](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/Client%20Download.png "Client Download Form")
- 4. The final step is to enter the Exercism API key which allows you to fetch, keep track of exercises, submit solution etc - ![Configure API Form](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/Configure%20API.png "Configure API Form")
- 5. That's it. Click `Finish` and you are done :0)
+# Step-By-Step Install Procedure    
+## License  
+ ![License Agreement](img/installLicense.png)  
+ This is the License Agreement form. To proceed with the installation accept the license agreement by clicking the `Next` button.  
+ 
+## Install Location  
+ ![Install Location](img/installLocation.png)  
+ Once you accept the agreement, you are presented with an Install Location screen. Browse or type the location of the folder where you would like to install the Exercism CLI.  Click the `Next` button when ready to proceed.  
+ 
+ You may use the `Browse` button to locate the folder in which to install the CLI.  
+ ![browse](img/browse.png)  
+Click the `Ok` button to accept your folder selection.
+
+## Download and Install  
+ ![Client Download Form](img/install.png)  
+ At this point the installer will proceed to download the latest version of the CLI for your OS architecture from the GitHub Repo for the Exercism CLI.  The downloaded ZIP file will automatically be unzipped into the folder you previously selected.  
+ ![download success](img/installFinish.png)  
+ Click the `Next` button once the process completes successfully.  
+ 
+## Configure CLI  
+ ![Configure API Form](img/configure.png)  
+ The final step is to enter the Exercism API key which allows you to fetch, keep track of exercises, submit solution etc.  This key is assigned to you when you initially sign up with Exercism.  Use the `I don't have or don't know my API key` link to retrieve your key as needed.  
+ Once you have input your API key and have input a `Solution Location`, click the `Configure` button.  This will invoke the Exercism CLI which will be passed the information you just entered.  The output of the CLI will appear in the box at the bottom of the screen (I have blocked out my information).  
```

Comment:
I need to get this stuff written for the CLI before I can answer this definitively, but I _think_ that the installer will be able to ditch the step that sets the token and workspace dir. The first time someone uses the CLI, they'll be guided through the steps of adding their token and choosing where to download things.